float64 timestamp
string newdb_name
bool data